src/test/java/uk/gov/mca/beacons/service/ExampleTest.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.mca.beacons.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+class ExampleTest {
```

Comment:
Can remove this test as I assume it was just used for testing the gradle integrationTest task worked as expected?